assets/css/style.css

```diff
@@ -18,869 +18,940 @@ h3,
-  height: auto;
-  margin: 0 auto 15px;
-  display: block;
-  text-align: center;
+    max-width: 100%;
+    height: auto;
+    margin: 0 auto 15px;
+    display: block;
+    text-align: center;
 }
 
 .content ul {
-  padding-left: 0;
-  margin-bottom: 20px;
+    padding-left: 0;
+    margin-bottom: 20px;
 }
 
 .content ul li {
-  padding-left: 20px;
-  position: relative;
+    padding-left: 20px;
+    position: relative;
 }
 
 .content ul li::before {
-  position: absolute;
-  content: '';
-  height: 8px;
-  width: 8px;
-  border-radius: 50%;
-  background: var(--primary-color);
-  opacity: .3;
-  left: 0;
-  top: 8px;
+    position: absolute;
+    content: '';
+    height: 8px;
+    width: 8px;
+    border-radius: 50%;
+    background: var(--primary-color);
+    opacity: .3;
+    left: 0;
+    top: 8px;
 }
 
 .list-styled li {
-  padding-left: 20px;
-  position: relative;
+    padding-left: 20px;
+    position: relative;
 }
 
 .list-styled li::before {
-  position: absolute;
-  content: '';
-  height: 8px;
-  width: 8px;
-  border-radius: 50%;
-  background: var(--primary-color);
-  opacity: .3;
-  left: 0;
-  top: 17px;
+    position: absolute;
+    content: '';
+    height: 8px;
+    width: 8px;
+    border-radius: 50%;
+    background: var(--primary-color);
+    opacity: .3;
+    left: 0;
+    top: 17px;
 }
 
 .post-meta {
-  color: var(--text-color);
-  font-style: italic;
-  font-size: 14px;
+    color: var(--text-color);
+    font-style: italic;
+    font-size: 14px;
 }
 
 blockquote {
-  font-size: 20px !important;
-  color: var(--text-color-dark);
-  padding: 20px 40px;
-  border-left: 2px solid var(--primary-color);
-  margin: 40px 0;
-  font-weight: bold;
-  background: var(--light-color);
+    font-size: 20px !important;
+    color: var(--text-color-dark);
+    padding: 20px 40px 0;
+    margin: 40px 0;
+    font-weight: 300;
+    ;
+}
+
+
+/* citation Parent div */
+
+.quoteblock {
+    background: #f5fdff;
+    border-left: 2px solid var(--primary-color);
+    box-sizing: border-box;
+}
+
+
+/* User Citation */
+
+.user-quote {
+    padding-bottom: 5px;
+}
+
+
+/* Citation image */
+
+.user-quote blockquote .imageblock {
+    border-radius: 50%;
+    height: 100px;
+    width: 100px;
+    overflow: hidden;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin: auto;
+}
+
+
+/* Citation Person name */
+
+.user-quote .attribution {
+    display: block;
+    text-align: center;
+    margin: auto;
+    margin-top: 2%;
+    color: var(--primary-color);
+    font-weight: 700;
+}
+
+
+/* Citation big quotes */
+
+.user-quote blockquote p::before {
+    font-family: "Font Awesome 5 Free";
+    content: "\f10d";
+    color: var(--primary-color);
+    font-weight: 700;
+    font-size: 50px;
+    padding: 3px 3px;
+    margin-right: 6px;
 }
 
-blockquote p {
-  margin-bottom: 0 !important;
+
+/* Citation text */
+
+.user-quote blockquote p {
+    text-align: center;
+    /*padding-bottom: 3rem;*/
```

Comment:
This looks much better than the original blockquote! 👍🏻 Also nice and tidy, easy to read.